raiden/blockchain/events.py

```diff
@@ -322,6 +322,9 @@ def uninstall_all_event_listeners(self):
         self.event_listeners = list()
 
     def add_event_listener(self, event_name, eth_filter, abi):
+        existing_listeners = [x.event_name for x in self.event_listeners]
```

Comment:
have you figured out why this happens? I didn't understand why a filter could be installed more than once

Reply:
I think this had happened only due to this `eth-tester` bug with duplicated filter events.
(I should've put assert instead of plain return in there as a security check.)